dask/dataframe/io/parquet/arrow.py

```diff
@@ -23,6 +23,7 @@
     _analyze_paths,
```

Comment:
Do we still *need* this import?  It seems that the remaining use of `_analyze_paths` could also use the new api.

Reply:
good pt, fixed